### DISCUSSION TITLE
⚽⚽⚽⚽Please add soccer teams to my list ⚽⚽⚽⚽

### DISCUSSION THREAD

**Discussion by ALORAMFER** [NAIST-SE/AttractingContributors#886] [2023 Attraction Contributions]
I need help with my soccer teams list.

Come to my fork:
https://github.com/ALORAMFER/AttractingContributors-SoccerTeams

Add to my README file.

I look forward to your contributions

**Comment by chariot-524** [comment]
あなたのリストにManchester City FCを追加するプルリクエストを送りました。
私のところにもプルリクエストしてもらえないでしょうか？あなたにとっての最高のゲームのタイトルを追加してほしいです。
https://github.com/chariot-524/AttractingContributors-Game

I made a pull request to add Manchester City FC to your list.
Could you add your best game to the list at the link below?
https://github.com/chariot-524/AttractingContributors-Game

**Reply by ALORAMFER** [reply]
Thank you so much! I will add it!
